spec/celluloid/supervision_group_spec.rb

```diff
@@ -58,5 +62,19 @@ class MyGroup < Celluloid::SupervisionGroup
 
       expect(supervisor[:example]).to be_a MyActor
     end
+
+    it "supports injections" do
+      group_klass = Class.new(Celluloid::SupervisionGroup) do
+        def foo
+          add(MyActor, as: :bar, injections: {
+            initialize: Proc.new { @name = :foo }
+          })
+        end
+      end
+      group = group_klass.run!
+      group.foo
+      sleep 0.01
```

Comment:
The sleeps need to be fixed - and they are (spec by spec). This one just isn't high priority.